package.json

```diff
@@ -49,7 +49,7 @@
     "@babel/preset-react": "7.18.6",
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-replace": "^2.3.1",
-    "@stripe/connect-js": "3.3.14-beta-1",
```

Comment:
I expected to see a yarn.lock bump associated with this PR too... is this missing?